docs/tutorial/tutorial-4.rst

```diff
@@ -81,10 +81,12 @@ for your existing bundled application:
 If Briefcase can't find the scaffolded template, it will automatically invoke
 `create` to generate a fresh scaffold.
 
-Now that we've updated the installer code, We can then run ``briefcase build``
+Now that we've updated the installer code, we can then run ``briefcase build``
 to re-compiled app, ``briefcase run`` to run the updated app, and ``briefcase
 package`` to repackage the application for distribution.
 
+(Remember that as noted in :doc:`Tutorial 3 <tutorial-3/index>`, for the tutorial we recommend running ``briefcase package`` with the ``--no-sign`` flag to avoid the complexity of setting up a code signing identity and keep the tutorial as simple as possible.)
```

Comment:
This is a good note - but it only applies to macOS; Windows and Linux users don't get the same option (at least, not yet... hopefully they will soon!). For now, at least, we should qualify the instructiona we give here.

Reply:
Thanks! I committed a small change to qualify the instructions to macOS only. Let me know if this works, or if something else is needed.